openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -823,26 +829,45 @@ const TeamDetailsV1 = ({
     }
 
     return (
-      <>
-        <div
-          className="tw-grid xxl:tw-grid-cols-4 md:tw-grid-cols-3 tw-gap-4"
-          data-testid="dataset-card">
-          {' '}
-          {ownData.map((dataset, index) => {
-            const Dataset = {
-              displayName: dataset.displayName || dataset.name || '',
-              type: dataset.type,
-              fqn: dataset.fullyQualifiedName || '',
-              id: dataset.id,
-              name: dataset.name,
-            };
-
-            return (
-              <UserCard isDataset isIconVisible item={Dataset} key={index} />
-            );
-          })}
-        </div>
-      </>
+      <div data-testid="table-container">
+        {assets.data.map((entity, index) => (
+          <div className="m-b-sm" key={`${entity.name}${index}`}>
+            <TableDataCard
+              database={entity.database}
+              databaseSchema={entity.databaseSchema}
+              deleted={entity.deleted}
+              description={entity.description}
+              fullyQualifiedName={entity.fullyQualifiedName}
+              id={`tabledatacard${index}`}
+              indexType={entity.index}
+              name={entity.name}
+              owner={entity.owner}
+              service={entity.service}
+              serviceType={entity.serviceType || '--'}
+              tags={entity.tags}
+              tier={
+                (
+                  entity.tier?.tagFQN ||
+                  getTierFromSearchTableTags(
+                    (entity.tags || []).map((tag) => tag.tagFQN)
+                  )
+                )?.split(FQN_SEPARATOR_CHAR)[1]
+              }
+              usage={entity.weeklyPercentileRank}
+            />
+          </div>
+        ))}
+        {assets.total > LIST_SIZE && assets.data.length > 0 && (
+          <NextPrevious
+            isNumberBased
+            currentPage={assets.currPage}
+            pageSize={LIST_SIZE}
+            paging={{} as Paging}
```

Comment:
Not sure why we are setting empty object as Paging

Reply:
paging object is not required in the component if `isNumberBased` prop is passed. The paging data is handle separately in our asset object.